zboss/doc/zboss_configuration.rst

```diff
@@ -5,9 +5,26 @@ ZBOSS configuration
 
 The ZBOSS stack is distributed as a set of precompiled linkable libraries.
 There are two variants of the libraries that support different Zigbee device roles, each variant having its own configuration.
-For a complete list of the ZBOSS configuration options, see the following files:
+For a complete list of the ZBOSS configuration options, used for building the production-ready version, see the following files:
 
-* :file:`zboss/include/osif/libzboss_config.h` - library for Coordinators and Routers
-* :file:`zboss/include/osif/libzboss_config.ed.h` - library for End Devices
+* :file:`zboss/v3.3/include/osif/libzboss_config.h` - library for Coordinators and Routers
+* :file:`zboss/v3.3/include/osif/libzboss_config.ed.h` - library for End Devices
+
+For a complete list of the ZBOSS configuration options, used for building the experimental (NCP-enabled) version of ZBOSS, see the following files:
+
+* :file:`zboss/v3.99/include/osif/libzboss_config.h` - library for Coordinators and Routers
+* :file:`zboss/v3.99/include/osif/libzboss_config.ed.h` - library for End Devices
 
 These libraries are used in the Zigbee protocol configuration in |NCS| when defining the Zigbee device role, as described in :ref:`nrf:zigbee_ug_configuration` in the |NCS| documentation.
+
+ZBOSS library variants (experimental version only)
+##################################################
+
+If the experimental version of ZBOSS is selected through the :option:`CONFIG_ZIGBEE_LIBRARY_EXPERIMENTAL` KConfig option,
+two additional configuration options may be enabled:
+
+* The application may link NCP commands implementation by enabling the :option:`CONFIG_ZIGBEE_LIBRARY_NCP_DEV` KConfig option.
+  For more information, please refer to the NCP firmware sample documentation.
```

Comment:
Needs link to NCP sample. @maciekfabia , would it make sense to point the NCP sample PR to this one?
Is this option already enabled for the sample?

Reply:
The NCP sample is out of scope of my task, so I thought it would be better to mention it here and update this section with the PR containing the sample.